tests/unit/dataactvalidator/test_d41_detached_award_financial_assistance_5.py

```diff
@@ -0,0 +1,78 @@
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+from dataactcore.models.domainModels import Zips
+
+_FILE = 'd41_detached_award_financial_assistance_5'
+
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "place_of_performance_code", "place_of_performance_zip4a"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ The provided PrimaryPlaceofPerformanceZIP+4 must be in the state specified by PrimaryPlaceOfPerformanceCode.
+        In this specific submission row, the first five digits are valid and located in the correct state, but the
+        last 4 are invalid."""
+
+    # XX00000 validates here because it passes as long as the zip is valid in that state, this is checked
```

Comment:
This comment is probably not needed. We aren't checking anything with this format.